pages/resources/user.mdx

```diff
@@ -812,6 +813,46 @@ This endpoint is only usable with an OAuth2 access token with the `account.globa
 | ------------ | ------- | ----------------------------------------- |
 | global_name? | ?string | The user's display name (1-32 characters) |
 
+<RouteHeader method="GET" url="/users/@me/avatars">
+  Get Recent Avatars
+</RouteHeader>
+
+Returns the user's recent avatars. Only 2 avatars will be returned for users without a Nitro subscription, otherwise 6 recent avatars will be returned.
```

Comment:
```suggestion
Returns the user's recent avatars. For premium users, the six most recent avatars will be returned. Otherwise, only two will be returned.
```